kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -176,6 +176,18 @@ data:
     [agent_settings.high_log_scale]
       enabled = false
 
+    # Retina Network Flow Logs throttle settings
+    # Controls the rate at which network flow log messages are processed. 
+    # throttle_rate: Amount of messages for the time.
+    # throttle_window: Amount of intervals to calculate average over. Default 300.
+    # throttle_interval: Time interval, expressed in "sleep" format. e.g 3s, 1.5m, 0.5h etc.
+    # throttle_print: Whether to print status messages with current rate and the limits to information logs.
+    # [agent_settings.networkflow_config]
```

Comment:
change from "networkflow_config" to "networkflow_logs_config"

Reply:
changed